include/internal/refcount.h

```diff
@@ -174,19 +174,19 @@ static __inline int CRYPTO_GET_REF(CRYPTO_REF_COUNT *refcnt, int *ret)
 
 static __inline int CRYPTO_UP_REF(CRYPTO_REF_COUNT *refcnt, int *ret)
 {
-    *ret = _InterlockedExchangeAdd(&refcnt->val, 1) + 1;
+    *ret = _InterlockedExchangeAdd((void *) &refcnt->val, 1) + 1;
```

Comment:
This looks wrong to me. In you're previous PR that was only merged in a few branches it was cast to `(long volatile *)`. This should probably do the same and merge it to all relevant branches.